### PR TITLE
{bio}[foss/2018b] bmtagger v3.101, SRPRISM v3.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/1310dfc86bc6e0c3889ad0ec391d8bfd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/863a2a0bf7b79b4dfe743c35943831d4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2012.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c38cf609b1105a9c96b0a2f48c4f3706 for a full test report.

File: easybuild/easyconfigs/b/bmtagger/bmtagger-3.101_fix-hardcoding.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+fix hardcoding of compiler command & flags, by passing down $CXX, $CXX and $CXXFLAGS
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/bmtagger/bmtagger-3.101_fix-hardcoding.patch]: you're passing down CFLAGS, not CXXFLAGS?

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 1 out of 3 (2 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/80bda5104c86dbe70aa229452dac7a04 for a full test report.

File: easybuild/easyconfigs/b/bmtagger/bmtagger-3.101_fix-hardcoding.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/b/bmtagger/bmtagger-3.101_fix-hardcoding.patch]: @wpoely86 The code is partially C & C++, and the flags are identical anyway...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f0fd1911142168b91efb06ae28bac79c for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/108d30756b792278e8613a5105b2da2b for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**